CONTRIBUTING.md

```diff
@@ -88,6 +107,136 @@ One of the best ways we can keep fastlane an approachable, stable, and dependabl
 
 Writing a custom action is an easy way to extend the capabilities of fastlane. Actions that make good candidates for inclusion in the fastlane codebase are **flexible** and apply to **many projects, teams, and development setups**. Before working to contribute your custom action to fastlane, consider whether it is likely to solve a problem that many developers have. If not, it can still provide value for your fastlane environment! Check out the documentation for creating [local action extensions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/README.md#extensions).
 
+## Developing
+
+When working on something, directly edit the Ruby files in the project folders. Make sure to switch to 2 spaces in your text editor.
```

Comment:
Yeah that's not super clear. I'll update. I don't want to spark the age old debate but we are using spaces right @KrauseFx?